src/participants.md

```diff
@@ -5,39 +5,56 @@ permalink: /participants/
 include: true
 ---
 
-## Principal Investigators:
+### Principle Investigator:
```

Comment:
I think we should change all of these to "Principal Investigators"

Reply:
This seems to be a deliberate distinction that Michael is making, so it's unclear why we would change that.

Reply:
Oops, I meant to suggest that it should be changed to "Principal Investigator." Using the singular certainly makes sense here.
On the other point, the vast majority of the sources I've consulted identify "Principle Investigator" as a misspelling of "Principal Investigator" and nothing more, except in the very rare case that the individual so entitled really is tasked with investigating principles -- so, like a theoretical physicist or something.

Reply:
Ah. Right, the spelling definitely needs to be fixed. I thought you were commenting on the new split between single PI and Associates